README.md

```diff
@@ -4,13 +4,19 @@
 Dynmap 3.x+ uses Gradle for building support for all platforms, with all resulting artifacts produced in the /targets directory
 
 To build, run:
+```
```

Comment:
These changes should really be in a separate PR as they're unrelated to the version string checks in Helper.java#38

Reply:
Also the title doesn't appear to accurately represent the changes proposed in the PR